### PR TITLE
Add project

### PR DESCRIPTION
**Pull request by malodev1234**
-I add files into my project 
- I add folder 
- I add readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!-- PROJECT SHIELDS -->
+
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues-open][issues-open-shield]][issues-url]
+[![Issues-closed][issues-closed-shield]][issues-url]
+[![Contributors][contributors-shield]][contributors-url]
+[![Framework][badge-framework]][framework-url]
+[![contributions welcome][contributions-welcome]][issues-url]
```
Comment (malodev1234) [inline README.md:9]: Please , can you add a  linter a benefique things inside your readme file . readme file must describe exactly what does yours project must do.you can changes theses lines by showing how to start with your project

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place.
```
Comment (malodev1234) [inline js/index.js:3]: so good so far but i would like to suggest you to use semantic tags.A semantic element clearly describes its meaning to both the browser and the developer.Examples of non-semantic elements: <div> and <span> - Tells nothing about its content. Examples of semantic elements: <form>, <table>, and <article> - Clearly defines its content.

**Review by malodev1234 — COMMENTED** [review]
please the title of your pull request must be descriptive. a descriptive message of commit can let you know what exactly your project is about
#Reques changes @

**Review by malodev1234 — COMMENTED** [review]
#no linter passed, please can you please add alinter in your project.linter let your code be clear

**Comment by malodev1234** [comment]
> -I add files into my project
> 
> * I add folder
> * I add readme file

Your code must be descritpi

**Comment by malodev1234** [comment]
> -I add files into my project
> 
> * I add folder
> * I add readme file

Your code must be descritptive

**Comment by joskalenda** [comment]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @malodev1234 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Github inline comment
✅ ... Polite communication

---------
### Optional suggestions

- [ ] Never close a pull request before it was approved, please. if you did it by mistake, you can still reopen it down from where you closed it in case it is still under review

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**